hexa/workspaces/templates/workspaces/mails/new_member.txt

```diff
@@ -0,0 +1,5 @@
+Your Have been added to an OpenHexa Workspace
```

Comment:
Would be nice to display the name of the workspace and the name of the user that sent the invitation.

Reply:
Yes, on it.